decidim-core/spec/lib/view_hooks_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  describe ViewHooks do
+    subject { described_class.new }
+
+    describe "register" do
+      it "saves the hooks in priority order" do
+        subject.register(:test_hook, priority: 3) do
+          "Lower priority"
+        end
```

Comment:
Missing newline

Reply:
Intended

Reply:
Why?

Reply:
Use inline blocks instead so we don't have to spill blood over it.